### PR TITLE
Add reference for JsLessThan and JsLessThanOrEqual

### PR DESCRIPTION
**Pull request by xiaoyinl**
This PR updates the documents for Microsoft/ChakraCore#3931.

/cc @liminzhu

### CODE REVIEW

File: reference/JsLessThan.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+Determine if one JavaScript value is less than another JavaScript value.
+### Syntax
+```
+STDAPI_(JsErrorCode)
```
Comment (liminzhu) [inline reference/JsLessThan.md]: Can you put it as below? Same goes for JsLessThanOrEqual.

```
CHAKRA_API
    JsLessThan(
        _In_ JsValueRef object1,
        _In_ JsValueRef object2,
        _Out_ bool *result);
```

**Comment by liminzhu** [comment]
Thanks again for adding docs!

**Comment by xiaoyinl** [comment]
@liminzhu Done. Thanks for the review!

**Comment by dilijev** [comment]
@liminzhu Any reason not to merge this PR now?

**Comment by dilijev** [comment]
@liminzhu ping

**Comment by liminzhu** [comment]
Oh no, we should just merge it.

**Pull request merged**